webroot/css/layouts/elements.css

```diff
@@ -271,7 +271,7 @@ html[dir="rtl"] .comment-sentence .text {
     margin: 0;
 }
 
-.comment.inappropriate .content {
+.comment.inappropriate > md-card-content > .content {
```

Comment:
I think it would make more sense to move the `inappropriate` class so that it only applies to the content instead of doing this direct child check.

There are also some other rules using `.inappropriate` [later in the file](https://github.com/Tatoeba/tatoeba2/blob/2f0d22913d48e3f61ba2be1108d0c6b676789e30/webroot/css/layouts/elements.css#L505), but I think those are obsolete. Might be best to delete them.

Reply:
Thanks for the review! I applied your suggestions.

Reply:
Nice :)